### DISCUSSION TITLE
What is the "Configuration protocol" for?

### DISCUSSION THREAD

**Discussion by prikrylm** [pascallanger/DIY-Multiprotocol-TX-Module#680] [Q&A] [answered]
I'm not able to find any information about the "Configuration" protocol (probably protocol no. 86).  
So, what is it for?
Maybe I'm just looking wrong. :)

**Comment by pascallanger** [comment, marked as answer]
It's explained here: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/tree/master/Lua_scripts#multiconfig

**Reply by prikrylm** [reply]
Nice, thank you!
